GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -1275,7 +1276,7 @@ bool AddArtificialRevisions(IReadOnlyList<ObjectId>? headParents = null)
                     CommitterEmail = userEmail,
                     Subject = ResourceManager.TranslatedStrings.Workspace,
                     ParentIds = new[] { ObjectId.IndexId },
-                    HasNotes = true
+                    Notes = ""
```

Comment:
```suggestion
                    Notes = null
```
It does not make a big difference. But it is a nullable property. So "no notes" should explicitly be indicated using `null`.

Reply:
> It looks like a three-state logic: null = no notes read yet, string.Empty = no notes for commit, some string = some notes exist.

Ahh okay, then this should be mentioned in the XML doc.

Reply:
> It looks like a three-state logic: null = no notes read yet, string.Empty = no notes for commit, some string = some notes exist.

Ahh okay, then this should be mentioned in the XML doc.